great-idea/index.html

```diff
@@ -14,10 +15,112 @@
   <![endif]-->
 </head>
 
-  <body>
+<body>
+  <div class="index-container">
+    <header>
+      <div class="nav-content-container">
+        <nav class="nav-bar">
+          <section class="nav-links">
+            <a href="services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </section>
+          <section class="nav-img">
+            <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+          </section>
+
+
+        </nav>
+      </div>
+
+      <div class="header-content-container">
+        <section class="header-content">
+          <div class="header-title">
+            <h1>
+              Innovation <br>
+              On <br>
+              Demand
+            </h1>
+
+            <button class="get-started">Get Started</button>
+
+          </div>
+          <div class="header-img">
+            <img src="img/header-img.png" alt="Image of a code snippet.">
+          </div>
+
+
+
+
+        </section>
+      </div>
+
+    </header>
+    <div class="top-content-container">
+      <section class="top-content">
+        <h2>Features</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+      <section class="top-content-2">
+        <h2>About</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+      </section>
+    </div>
```

Comment:
here in this area you want to switch your divs and sections. you will have one section that will contain the 2 divs
so your section will be the "top-content-container" class and the div will be both the "top-content" and "top-content-2"